1_preprocessing_data/README.md

```diff
@@ -0,0 +1,26 @@
+# 1. Preprocessing NF1 Data
+
+In this module, we present our pipeline for preprocessing the NF1 Schwann Cell data.
+
+## Illumination Correction
+
+To correct for illumination issues within the NF1 data, we use the BaSiC method that was established in an article by [Peng et al.](https://doi.org/10.1038/ncomms14836).
+We specifically use the Python implementation of this method, called [PyBaSiC](https://github.com/peng-lab/BaSiCPy).
```

Comment:
It's great to see the citations here! You might also want to add a sentence describing why illumination correction is important.